From 05391f98616decf9977a310b2e32b67eb8820338 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 30 Mar 2022 15:22:59 +0200
Subject: [PATCH 058/841] refactor(stm32mp13-fdts): remove secure-status for
 PMIC

PMIC driver now forces DT_SECURE status for its node and its parent
i2c node. The secure-status is replaced with status = "okay".

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 13c2332fb09fb49dbe18ed0d8197d492bc7d5927 refactor(stm32mp13-fdts): remove secure-status for PMIC]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I515e31889f2ecbe7bf045239778000f8751b2a50
---
 fdts/stm32mp135f-dk.dts | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/fdts/stm32mp135f-dk.dts b/fdts/stm32mp135f-dk.dts
index 120469203..7a03fe141 100644
--- a/fdts/stm32mp135f-dk.dts
+++ b/fdts/stm32mp135f-dk.dts
@@ -70,15 +70,13 @@
 	i2c-scl-rising-time-ns = <185>;
 	i2c-scl-falling-time-ns = <20>;
 	clock-frequency = <400000>;
-	status = "disabled";
-	secure-status = "okay";
+	status = "okay";
 
 	pmic: stpmic@33 {
 		compatible = "st,stpmic1";
 		reg = <0x33>;
 
-		status = "disabled";
-		secure-status = "okay";
+		status = "okay";
 
 		regulators {
 			compatible = "st,stpmic1-regulators";
-- 
2.47.2

